src/OpenTelemetry.Api/Metrics/GaugeHandle.cs

```diff
@@ -14,27 +14,30 @@
 // limitations under the License.
 // </copyright>
 
+using OpenTelemetry.Context;
 using OpenTelemetry.Trace;
 
 namespace OpenTelemetry.Metrics
 {
     /// <summary>
-    /// Handle to the gauge with the defined <see cref="LabelSet"/>.
+    /// Handle to the metrics gauge with the defined <see cref="LabelSet"/>.
     /// </summary>
-    /// <typeparam name="T">The type of the Gauge. Only long and double are supported now.</typeparam>
-    public struct GuageHandle<T>
+    /// <typeparam name="T">The type of gauge. Only long and double are supported now.</typeparam>
+    public abstract class GaugeHandle<T>
         where T : struct
     {
         /// <summary>
-        /// Sets the value of the guague.
+        /// Add, Set or Record the value of the counter handle.
         /// </summary>
+        /// <param name="context">the associated span context.</param>
+        /// <param name="value">value by which the gauge handle should be set.</param>
+        public abstract void Set(in SpanContext context, T value);
+
         /// <summary>
-        /// Sets the value of the guague handle.
+        /// Add, Set or Record the value of the counter handle.
```

Comment:
you can restore old comment - Sets the value of the gauge handle.

Reply:
```suggestion
        /// Sets the value of the guague handle.
```